src/arizona_websocket.erl

```diff
@@ -66,50 +81,63 @@ websocket_init({Params, {Mod, Fun, Opts}}) ->
     send(init, {init, self()}),
     {Events, State}.
 
+-spec websocket_handle(Event, State1) -> {Events, State2}
+    when Event :: {text, binary()},
+         Events :: cowboy_websocket:commands(),
+         State1 :: state(),
+         State2 :: state().
 websocket_handle({text, Msg}, #{sockets := Sockets} = State) ->
-    ?LOG_INFO("[WebSocket] handle: ~p~n", [Msg]),
     {Target, Event, Payload} = decode_msg(Msg),
     Socket0 = maps:get(Target, Sockets),
-    View = maps:get(view, Socket0),
-    {noreply, Socket1} = arizona_live_view:handle_event(View, Event, Payload, Socket0),
-    Socket = case maps:get(changes, Socket1) of
+    View = arizona_socket:get_view(Socket0),
+    Socket1 = arizona_live_view:handle_event(View, Event, Payload, Socket0),
+    Socket = case arizona_socket:get_changes(Socket1) of
         Changes when map_size(Changes) > 0 ->
             Tpl = maps:get(template, State),
-            Assigns = maps:get(assigns, Socket1),
+            Assigns = arizona_socket:get_assigns(Socket1),
             Patch = arizona_tpl_render:render_target(Target, Tpl, Changes, Assigns),
             arizona_socket:push_event(~"patch", [Target, Patch], Socket1);
         #{} ->
             Socket1
     end,
-    Id = maps:get(id, Socket),
-    {[{text, json:encode(maps:get(events, Socket))}],
+    Id = arizona_socket:get_id(Socket),
+    {[{text, json:encode(arizona_socket:get_events(Socket))}],
         State#{sockets => Sockets#{Id => arizona_socket:prune(Socket)}}}.
 
+-spec websocket_info(Event, State1) -> {Events, State2}
+    when Event :: {text, binary()},
```

Comment:
```suggestion
-spec websocket_info(Info, State1) -> {Events, State2}
    when Info :: any(),
```
[Reference](https://github.com/ninenines/cowboy/blob/b77dd29133365a11a4884af8f0fe432981d3f8a3/src/cowboy_websocket.erl#L62).
> I prefer using `term()` instead of `any()`, it sounds "more Erlang" to me.

Reply:
I'm Ok to change it, but we're only using `{text, _}`, for the time being, and it helps readability and maintenance.

Reply:
9b0d6ef